exampleCourse/questions/template/string-input/random/question.html

```diff
@@ -0,0 +1,19 @@
+<pl-question-panel>
+  <p>
+    Convert {{ params.a }} from decimal representation (base 10) to binary representation (base 2).
+  </p>
+
+  <p>
+    Your answer will be interpreted as a string. Do not include any leading zeros. For example, if
+    the answer is 11010, the binary number 0011010 will be <i>marked</i>
+    as incorrect, even though they both represent the same integer number.
+  </p>
+</pl-question-panel>
+
+<pl-string-input
```

Comment:
Should probably use `pl-integer-input` with the base set to binary instead. That will also make the backend simpler.

Reply:
That's a reasonable suggestion. Do you have any ideas for what a good demo of `<pl-string-input>` would be?

Reply:
An interesting use might be something like solving a crossword puzzle? Something more language based / abstract. It's actually a little nontrivial to think of examples because PL leans heavily into the sciences and any numerical quantity is handled better by other elements.

Reply:
Maybe something chemical related? "What is the chemical formula for {water,methane,etc}"

Reply:
I like the chemical formula idea, but we'd need to explain that you don't need subscripts (e.g., water is written `H2O`, not H<sub>2</sub>O).

Some other ideas:
1. Something explicitly related to language? "What is the plural of <mouse/child/goose/...>?"
2. Probably too hard, but "Give a string that matches the regex /((a?b){2}b)+/"

Reply:
I switched to the word pluralization example in 33f1580.